ush/generate_FV3LAM_wflow.sh

```diff
@@ -609,12 +620,7 @@ print_info_msg "Generating an alternate simple launch script
 ${EXPTDIR}/run_rocoto.sh"
 
 echo "#!/bin/bash" > ${EXPTDIR}/run_rocoto.sh
-if [[ "${MACHINE,,}" == "wcoss2" ]] ; then
-  echo "module use /apps/ops/test/nco/modulefiles" >> ${EXPTDIR}/run_rocoto.sh
-  echo "module load core/rocoto/${rocoto_ver}" >> ${EXPTDIR}/run_rocoto.sh
-else
-  echo "module load rocoto" >> ${EXPTDIR}/run_rocoto.sh
-fi
+echo "module load rocoto" >> ${EXPTDIR}/run_rocoto.sh
```

Comment:
The latest dev-sci branch is overwritten at this place. Please get back to what's in the latest dev-sci branch.

Reply:
@EdwardColon-NOAA Need to get back Line 612-617 from the left side (or the dev-sci branch)